### DISCUSSION TITLE
没有电脑只能这样直面天命了

### DISCUSSION THREAD

**Discussion by 15666851105** [share121/inter-knot#3313] [其他]
![IMG_20240820_152721](https://github.com/user-attachments/assets/215958d6-fcf1-47e1-99c1-a40cc973cfd7)

**Comment by stormcat-efforia** [comment]
抢到限定周边了吗

**Reply by 15666851105** [reply]
没有，周边早被抢光了
<img src="https://img-static.mihoyo.com/communityweb/upload/1b5600e9d6d16de86447348909796aa4.png" alt="安比 哭哭" width="75" >

**Reply by 15666851105** [reply]
人实在是太多了
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
